src/plugins/dashboard/public/application/app.scss

```diff
@@ -0,0 +1,11 @@
+.dshAppContainer {
+  display: flex;
+  flex-direction: column;
+  flex-grow: 1;
+}
+
+#dashboardViewport {
```

Comment:
The #dashboardViewport appears to be an empty <div> element now. Will you modify it or you just forget to remove it? loll

Reply:
`#dashboardViewport` is controlling the style for <div id="dashboardViewport>. It is not an empty <div>, it got rendered here: 

https://github.com/opensearch-project/OpenSearch-Dashboards/blob/ef248f479936b69578079bc58cf0963f763b5cc5/src/plugins/dashboard/public/application/utils/use/use_editor_updates.ts#L20

https://github.com/opensearch-project/OpenSearch-Dashboards/blob/ef248f479936b69578079bc58cf0963f763b5cc5/src/plugins/dashboard/public/application/utils/use/use_editor_updates.ts#L63